### PR TITLE
release: v1.9.8 - update contributors

### PR DESCRIPTION
**Pull request by frdeboffles**
I updated the contributions from `frd` to use my name and the right link to my github account (my bad authoring my commit using my company's user info).
I also swapped the `v1.9.7.md` and `v1.9.8.md` files since 1.9.8 content was in `v1.9.7.md` and vice versa.

### CODE REVIEW

**Comment by frdeboffles** [comment]
Had to rebase after the 1.9.9 release

**Comment by edsiper** [comment]
cc: @patrick-stephens

**Comment by patrick-stephens** [comment]
Apologies, will pick this up next chance I get.

File: content/announcements/v1.9/v1.9.8.md [hunk 1]
```diff
@@ -2,9 +2,9 @@
 title: 'v1.9.8'
 description: 'We provides the means for the collection, organization and computerized retrieval of knowledgeand Lightweight Data Forwarder for Linux, BSD, macOS and Windows.'
```
Comment (patrick-stephens) [inline content/announcements/v1.9/v1.9.8.md:3]: Same typos:
```suggestion
description: 'We provide the means for the collection, organization and computerized retrieval of knowledge and Lightweight Data Forwarder for Linux, BSD, macOS and Windows.'
```suggestion

File: content/announcements/v1.9/v1.9.7.md [hunk 2]
```diff
@@ -1,60 +1,50 @@
 ---
-title: 'v1.9.8'
+title: 'v1.9.7'
 description: 'We provides the means for the collection, organization and computerized retrieval of knowledgeand Lightweight Data Forwarder for Linux, BSD, macOS and Windows.'
```
[the remark this answers is not in the document]
Comment (frdeboffles) [reply content/announcements/v1.9/v1.9.7.md:3]: I did not do a copy paste but a `git mv` to switch the 2 files... Unfortunately, github saw that as a full rewrite...
Anyway, I'll fix those while I'm here. 
Thank you for reviewing

**Review by patrick-stephens — APPROVED** [review]

**Pull request merged**